Add iconRight prop to TextButton.

lavelle opened the pull request.

webbower commented on packages/thumbprint-react/components/UIAction/plain.jsx:
```diff
@@ -9,17 +9,28 @@ import styles from './plain.module.scss';
+        );
+    }
+
+    if (props.iconRight) {
```
Should `iconLeft` and `iconRight` be mutually exclusive props? If so, a note about that should probably be added to the docs for the components

webbower commented:
Also, should probably at least add tests for the change(s) you add, including tests for all permutations of `iconLeft` and `iconRight` props

lavelle commented:
- Fixed the mutually exclusive issue
- Add test
- Fixed docs
- Update changelog.

Not gonna add to any other components. want to keep this change simple.
